cogs/csgo.py

```diff
@@ -139,6 +147,16 @@ async def pug(self, ctx: commands.Context, *args):
 
             emoji_remove = []
 
+            player_veto = []
+            if self.bot.match_size == 2:
+                player_veto = [1, 1]
+            for i in range(self.bot.match_size - 2):
+                if i == 0 or i == self.bot.match_size - 3:
+                    player_veto.append(1)
+                elif i % 2 == 0:
+                    player_veto.append(2)
+            player_veto = player_veto + [1, 1]
```

Comment:
If the match size is 2 the player_veto would be `[1,1,1,1]`, not sure if this is is intended or not

Reply:
This is intended. I add [1, 1] to end of every veto for coaches. It'll never be used though in the queue and will only be used if you run .pug with more people than the match size